designer/package.json

```diff
@@ -22,8 +22,8 @@
     "test": "jest --color --coverage --verbose",
     "test:watch": "jest --color --watch",
     "client:watch": "NODE_ENV=development npm run build:client -- --watch",
-    "server:watch": "nodemon --exec tsx --enable-source-maps ./server/src",
-    "server:debug": "nodemon --exec tsx --enable-source-maps --inspect ./server/src",
+    "server:watch": "nodemon --exec tsx --env-file=../.env --enable-source-maps ./server/src",
+    "server:debug": "nodemon --exec tsx --env-file=../.env --enable-source-maps --inspect ./server/src",
```

Comment:
Just double checking we're moving the **.env** file to ~`designer/.env`~ `.env` at project root?

We previously scoped it to the npm workspace

```js
configDotenv({
  path: [resolve(dirname(configPath), '../../.env')]
})
```

I'm happy to move it, just need to let everyone know

Reply:
Might have reached the point where we move the default flags into **nodemon.json**? [See docs](https://github.com/remy/nodemon/blob/master/doc/sample-nodemon.md)

Reply:
I'll leave it in `designer`, I misunderstood where it currently was. 

Good shout on nodemon, I'll move it over.

Reply:
Cheers for that

Presume it'll still work with Nodemon merging the default flags with `--inspect` for `server:debug`

Don't worry about it if not

Reply:
Got it working, mind giving the config a quick glance over?

[nodemon.json](https://github.com/DEFRA/forms-designer/pull/122/files#diff-1c13f4cb1ae8251360a932c450eb6107b79754e84f57c5e7a54706fa1dfcc0e5)
[package.json](https://github.com/DEFRA/forms-designer/pull/122/files#diff-24a3bf952e8ace70037412955431f169d560b3c29d41aa85a7351a9416fec173)

Reply:
Just mentioned on Slack, it's working but `--inspect` gets placed after the Node.js (or TypeScript) path

You'd normally see this line logged to the terminal:

```console
Debugger listening on ws://127.0.0.1:9229/5396a022-ac92-4405-a131-0a8894b9f2fa
For help, see: https://nodejs.org/en/docs/inspector
```

Thankfully we can swap `exec` → `execMap` and move the path back into **package.json**

```json
"execMap": {
  "js": "node --enable-source-maps --env-file=.env",
  "ts": "node --enable-source-maps --env-file=.env --import tsx"
}
```

As an added bonus, it'll now skip `--import tsx` for plain old Node.js input when needed

Pushed it up in https://github.com/DEFRA/forms-designer/pull/122/commits/581376e46e72a7a48da35b965819a98776cf81e8

Reply:
Amazing!